### DISCUSSION TITLE
react-hook-forms integration example

### DISCUSSION THREAD

**Discussion by AlexeyUryvaev** [gpbl/react-day-picker#1441] [Support] [answered]
After switching to version 8, the standard Controller of react-hook-forms v7 does not work. Maybe someone has an example of such integration with an updated component. Thanks

**Comment by gpbl** [comment, marked as answer]
@AlexeyUryvaev would you mind to share something in https://codesandbox.io/s/react-day-picker-8-vl4ry so we can investigate? Thanks!

**Reply by AlexeyUryvaev** [reply]
It's all ok now. It is necessary to add  to the list of UpdatedProps "weekStartsOn" instead of "firstDayOfWeek". And also change in import instead of "import DayPicker from "react-day-picker"" ->" import { DayPicker } from "react-day-picker""; Thnx
